.travis.sh

```diff
@@ -78,8 +78,11 @@ case $TEST_PACKAGE in
 
                 #http://askubuntu.com/questions/204510/how-to-install-python-2-5-4
                 sudo apt-add-repository -y ppa:fkrull/deadsnakes
```

Comment:
(I'm a deadsnakes maintainer)

please instead use the new `ppa:deadsnakes` (or `ppa:deadsnakes/ppa` if you prefer the verbose name)